docs/appguide.rst

```diff
@@ -68,16 +70,16 @@ any change in its lifecycle.
         BACKGROUND -> START [ label=" __del__()   " ];
         BACKGROUND -> ACTIVE [ label=" foreground()   " ];
         ACTIVE -> BACKGROUND [ label=" background()   " ];
-        ACTIVE -> GO_TO_CLOCK [ label=" sleep() -> False   " ];
-        GO_TO_CLOCK -> BACKGROUND [ label=" background()   " ];
-        ACTIVE -> SLEEPING [ label=" sleep() -> True   " ];
-        SLEEPING -> ACTIVE [ label=" wake()   " ];
+        ACTIVE -> RET_TO_CLOCK [ label=" sleep() -> False   " ];
+        RET_TO_CLOCK -> BACKGROUND [ label=" background()   " ];
+        ACTIVE -> SYSTEM_SLEEPING [ label=" sleep() -> True   " ];
+        SYSTEM_SLEEPING -> ACTIVE [ label=" wake()   " ];
 
         START [ shape=box ];
         BACKGROUND [ shape=box, style=rounded ]
         ACTIVE [ shape=box, style=rounded ]
-        SLEEPING [ shape=box, style=rounded ]
-        GO_TO_CLOCK [ label="GOTO ClockApp" ];
+        SYSTEM_SLEEPING [ shape=box, style=rounded ]
```

Comment:
This diagram is the an application state diagram not a system state diagram. In other words it diagrams the state an app can be in and what app entry points are called to change application state.

SYSTEM_SLEEPING is not a valid state for an application to be in (an app is an app, the system is the system, only the system can be in a system sleep state).

Reply:
Alright, what's the best way to remove this? Should I just do a git revert ?